api/nodes/views.py

```diff
@@ -11,7 +11,10 @@
 from .serializers import NodeSerializer, NodePointersSerializer, NodeFilesSerializer
 from api.users.serializers import ContributorSerializer
 from .permissions import ContributorOrPublic, ReadOnlyIfRegistration, ContributorOrPublicForPointers
-
+from api.base.utils import node_token_creator, absolute_reverse
+from website.language import BEFORE_DELETE
+from rest_framework import serializers
+from django.utils.translation import ugettext_lazy as _
```

Comment:
Imports from a third-party library, in this case `rest_framework` and `django`, should be in the second import section.

ref: http://cosdev.readthedocs.org/en/latest/style_guides/python.html#imports

I personally prefer that imports be in alphabetical order within those sections, but that's not part of the styleguide, and not something I'd ask you to change. It's just my OCD nature showing through :)